src/order/category/NonemptyFinLinOrd.lean

```diff
@@ -9,6 +9,9 @@ import order.category.LinearOrder
 /-! # Nonempty finite linear orders
 
 Nonempty finite linear orders form the index category for simplicial objects.
+
+Note: `FinDistribLattice` is NOT a subcategory of `NonemptyFinLinOrd` because its morphisms preserve
+`⊥` and `⊤`.
```

Comment:
Why does this make it not a subcategory? It seems like this just means it's not a full subcategory - don't you still have a faithful functor `FinDistribLattice` to `NonemptyFinLinOrd`? Or does nonemptiness cause issues...

Reply:
How come? Finite distributive lattices are not linear orders. But you're right that I wrote this the wrong way around.

Reply:
All I mean is that your explanation doesn't match the claim here.

Reply:
And now?

Reply:
The issue has not been fixed. Extra morphisms present in the target category is not a restriction to being a subcategory.

Reply:
Read more carefully. There are extra morphisms in the **source** category.

Reply:
I'm sorry, what are you suggesting I read more carefully? The claim here is that "`FinDistribLattice` is NOT a subcategory of `NonemptyFinLinOrd` because its morphisms preserve `⊥` and `⊤`." is it not? And morphisms in `FinDistribLattice` do preserve `⊥` and `⊤`, while morphisms of `NonemptyFinLinOrd` don't, so there are extra morphisms in the target category. I do not believe I am misreading anything here, could you clarify what precisely you think I am misreading?

Reply:
Your comment is outdated. Refresh Github and have a look at the diff.

Reply:
Ah I see. But you are still missing the functor from `NonemptyFinLinOrd` to `DistribLattice`, no?

Reply:
It comes from `has_forget₂ NonemptyFinLinOrd LinearOrder`!